### PR TITLE
LPD-43304 | master

### PR DESCRIPTION
**Pull request by fmarcio**
Jira ticket https://liferay.atlassian.net/browse/LPD-43304

### CODE REVIEW

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/hoc/segment/WithBaseEdit.jsx [hunk 1]
```diff
@@ -52,7 +57,9 @@ export default WrappedComponent => {
 		};
 
 		state = {
-			onDelete: false
+			onDelete: false,
+			show: false,
+			showTooltip: false
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/hoc/segment/WithBaseEdit.jsx:62]: It is not needed.

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/pages/List.tsx [hunk 2]
```diff
@@ -308,6 +308,7 @@ export const List: React.FC<IListProps> = ({
 			label: Liferay.Language.get('dynamic-segment')
 		},
 		{
+			disabled: true,
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/pages/List.tsx:311]: Please, rename it to `deprecated`.

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/base-page/Header.tsx [hunk 3]
```diff
@@ -140,9 +141,20 @@ const PageActions: React.FC<IPageActionsProps> = ({
 					</ClayButton>
 				}
 			>
-				{actions.map(({label, ...props}) => (
+				{actions.map(({disabled, label, ...props}) => (
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/base-page/Header.tsx:144]: It should be deprecated.

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/base-page/Header.tsx (hunk 3 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/base-page/Header.tsx]: @interaminense done

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/hoc/segment/WithBaseEdit.jsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/hoc/segment/WithBaseEdit.jsx]: @interaminense done

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**